tlv/onion_types.go

```diff
@@ -0,0 +1,15 @@
+package tlv
+
+const (
+	// AmtOnionType is the type used in the onion to refrence the amount to
+	// send to the next hop.
+	AmtOnionType Type = 2
```

Comment:
Yeah I had this in a few other places, then settled on this since the package doesn't import any other internal packages other than for benchmarks. I ran into an import cycle earlier on when it was in the switch as an example.